src/main.js

```diff
@@ -699,7 +699,7 @@ function initializeAfterAppReady() {
 
   mainWindow = createMainWindow(config.data, {
     hideOnStartup,
-    trayIconShown: process.platform === 'win32' || config.showTrayIcon,
+    trayIconShown: config.showTrayIcon,
```

Comment:
changing this line, allows win32 users to not show the icon. Was that intended?

Reply:
oh I understood that since showTrayIcon will be initially true this logic will affect the windows users as well